.github/workflows/push.yml

```diff
@@ -2,6 +2,7 @@ name: PHP Checks
 
 on:
   - push
+  - pull_request
```

Comment:
Should be : 

```yaml
on:
  pull_request:
  push:
    branches:
      - master
```

if you don't want the CI to run twice on PR that comes from another branch in the repo.

Reply:
Thx for the suggestion. Adjusted